hummingbot/connector/derivative/binance_perpetual/binance_perpetual_derivative.py

```diff
@@ -259,7 +259,7 @@ async def create_order(self,
                                               add_timestamp = True,
                                               is_signed=True)
             exchange_order_id = str(order_result["orderId"])
-            tracked_order = self._in_flight_orders.get(order_id)
+            tracked_order = self._in_flight_orders.get(order_id, None)
             if tracked_order is not None:
                 self.logger().info(f"Created {order_type.name.lower()} {trade_type.name.lower()} order {order_id} for "
                                    f"{amount} {trading_pair}.")
```

Comment:
In the next line (266) the exchange_order_id should be updated in the inflight order by sending the `update_exchange_order_id` function, in case some other process is waiting for that information by using the internal event.